CONTRIBUTING.md

```diff
@@ -1,38 +1,76 @@
-## Dependencies
+# Contributing to MyDuckServer
 
-Install the required package using pip:
+Thank you for contributing to MyDuckServer! This guide will help you set up the development environment and run the server.
 
-```bash
-pip3 install "sqlglot[rs]"
-```
+## Prerequisites
 
-### MacOS Specific Instructions
+Before getting started, ensure that the following dependencies are installed:
 
-If you encounter the following error on MacOS:
+1. **Go**  
+   Download and install the latest version of Go by following the [official installation guide](https://go.dev/doc/install).
 
-```
-error: externally-managed-environment
+2. **Python and `sqlglot[rs]` package**  
+   MyDuckServer depends on the `sqlglot[rs]` package, which can be installed using `pip3`. You have two options for installation:
 
-× This environment is externally managed
-```
+    - **Global installation** (use with caution as it may affect system packages):
+      ```bash
+      pip3 install "sqlglot[rs]" --break-system-packages
+      ```
 
-You can resolve it by adding the `--break-system-packages` flag:
+    - **Installation inside a virtual environment** (recommended for isolated environments):
+      ```bash
+      python3 -m venv myduck_venv
+      source myduck_venv/bin/activate
```

Comment:
This will create a `myduck_venv` directory in the current directory. Let's instruct the contributor to create it under the home directory (instead of some random place):

```
mkdir -p ~/venv
python3 -m venv ~/venv/myduck
source ~/venv/myduck/bin/activate
```

Reply:
fixed